helix-view/src/editor.rs

```diff
@@ -856,8 +844,8 @@ pub struct Editor {
     pub auto_pairs: Option<AutoPairs>,
 
     pub idle_timer: Pin<Box<Sleep>>,
-    pub last_motion: Option<Motion>,
-
+    #[allow(clippy::complexity)]
+    last_motion: Option<Arc<Box<dyn Fn(&mut Editor)>>>,
```

Comment:
Why not make a `type` for it?

Reply:
Only used once 🤷

Reply:
You can probably directly use `Arc<dyn Fn(&mut Editor)>` instead of wrapping a `Box` in an `Arc`

Reply:
You're right, thanks :)

Reply:
I agree with pickfire here: even in cases like this where it's a private field in a struct and only used once we always make another type. See `helix-term/src/ui/picker.rs` for example.

Reply:
Would a type alias be enough? https://github.com/helix-editor/helix/pull/6078/commits/57eab67f07169fa03361191954bf9101207404f0

Reply:
Yep I think that's fine. The newtype it was before was probably not necessary